### PR TITLE
added alerte_niger flavor

### PR DESCRIPTION
**Pull request by RishJ08**

### CODE REVIEW

File: Makefile
Comment (mrsarm) [inline Makefile:4]: Please rollback this change, you can still build the flavor from the command line passing your flavor as argument, eg.:

```make flavor=alerte_nigerWebView assemble```

File: build.gradle
Comment (mrsarm) [inline build.gradle:11]: I'm addressing the issues with the Gradle plugin in this PR: https://github.com/medic/cht-android/pull/202

So after merge it you will need to rollback these changes and get the fix from there.

File: gradle/wrapper/gradle-wrapper.properties
Comment (mrsarm) [inline gradle/wrapper/gradle-wrapper.properties:1]: Same about Gradle issues, we will fix it in other PR.

**Review by mrsarm — CHANGES_REQUESTED** [review]
@RishJ08 great job ! I leave some minor changes, and as I mention in the comments, we will need to wait for another PR that you will need to rebase once ready before merge this.

Also because there are changes in the Play Store that require changes in the way we sign the app, we will need to generate a set of key for signing this app, but that is also something under development so maybe we can do that in a later PR.

CC @Fatoufall

File: build.gradle
Comment (RishJ08) [reply build.gradle]: modified

File: gradle/wrapper/gradle-wrapper.properties
Comment (RishJ08) [reply gradle/wrapper/gradle-wrapper.properties]: modified

File: Makefile
Comment (RishJ08) [reply Makefile]: rolled back

**Comment by mrsarm** [comment]
@RishJ08 I have merged the https://github.com/medic/cht-android/pull/202 into master, so you need now to rebase or merge master into this branch. The changes allowed CI to work again, so after you add those changes into your PR I'll be able to hit the _"Aprove and run"_ button and get the green light to merge it:

GH Appr

**Review by mrsarm — APPROVED** [review]

**Pull request merged**
